docs/source/whats_new.rst

```diff
@@ -1,3 +1,54 @@
+What's New in cartopy 0.15
+==========================
+
+:Release: 0.15.0
+:Date: 1st February 2017
+
+Features
+--------
+
+* The :class:`cartopy.crs.Mercator` class now allows a ``latitude_true_scale``
+  to be specified. 
+
+* A ``tiles`` url can now be passed directly to the
+  :class:`cartopy.io.img_tiles.GoogleTiles` class. 
+
+* The :meth:`~cartopy.mpl.geoaxes.GeoAxes.background_img` method has been
```

Comment:
Yeah I did have more info, but I found myself just repeating the docstring. I'm happy to put it back thought.